test/integration/flows/code-flow.spec.ts

```diff
@@ -16,6 +17,15 @@ const AUTH_PARAMS = {
   state: "state",
 };
 
+function getOTP(email: EmailOptions) {
+  const codeEmailBody = email.content[0].value;
+  // this gets the space before so we don't match CSS colours
```

Comment:
copy-pasted from login2 :smile:   it's actually matching for *NOT* a hash before so it doesn't match CSS colours
```suggestion
  // this ignores number prefixed by hashes so we don't match CSS colours
```